packages/blocks/src/root-block/widgets/doc-remote-selection/doc-remote-selection.ts

```diff
@@ -205,6 +216,24 @@ export class AffineDocRemoteSelectionWidget extends WidgetComponent {
     return this.host.selection;
   }
 
+  private get config(): DocRemoteSelectionConfig {
+    const config =
+      this.std.spec.getConfig('affine:page')?.docRemoteSelectionWidget ?? {};
+
+    return {
+      blockSelectionBackgroundTransparent: block => {
+        return (
+          matchFlavours(block, [
+            'affine:code',
+            'affine:database',
+            'affine:image',
+          ]) || !/affine:embed-*/.test(block.flavour)
```

Comment:
Maybe we can add an "isVoid" field in the "Schema" to differentiate between editable and non-editable blocks.

Or can we render remote block selection and remote text selection in two ways?

Reply:
I think we can do that as a new feature, for a bug fix, the changes look good to me.